packages/web-twig/src/Resources/components/Accordion/README.md

```diff
@@ -0,0 +1,198 @@
+                {% embed "@spirit/accordionContent.twig" with { props: {
+                  id: 'AccordionItemExampleContent',
+                  labelId: 'AccordionItemExampleHeader',
+                  parent: 'AccordionExample',
+                  isOpen: true,
+                } } %}
+                    {% block content %}
+                        Accordion Content
+                    {% endblock %}
+                {% endembed %}
+            {% endblock %}
+        {% endembed %}
+    {% endblock %}
+{% endembed %}
+```
+
+## API
+
+The Accordion itself consists of many components which cannot be used independently.
+
+### Accordion
+
+| Prop name     | Type     | Default     | Required | Description              |
+| ------------- | -------- | ----------- | -------- | ------------------------ |
+| `id`          | `string` | `null`      | no/yes   | Optional, depends on ... |
+| `class`       | `string` | `null`      | no       | Additional class name    |
+| `elementType` | `string` | `'section'` | no       | Custom element type      |
+
+### AccordionItem
+
+| Prop name     | Type     | Default     | Required | Description           |
+| ------------- | -------- | ----------- | -------- | --------------------- |
+| `id`          | `string` | `null`      | yes      | Accordion Item ID     |
+| `class`       | `string` | `null`      | no       | Additional class name |
+| `elementType` | `string` | `'article'` | no       | Custom element type   |
+
+### AccordionHeader
+
+| Prop name     | Type     | Default | Required | Description              |
+| ------------- | -------- | ------- | -------- | ------------------------ |
+| `id`          | `string` | `null`  | yes      | AccordionHeader ID       |
+| `class`       | `string` | `null`  | no       | Additional class name    |
+| `for`         | `string` | `null`  | yes      | Target element selector  |
+| `elementType` | `string` | `'h3'`  | no       | Custom element type      |
+| `slot`        | `HTML`   | ``      | no       | Side slot in the header  |
+| `isOpen`      | `string` | `false` | no       | Whether is item selected |
+
+### AccordionContent
+
+| Prop name | Type     | Default | Required | Description                                                         |
+| --------- | -------- | ------- | -------- | ------------------------------------------------------------------- |
+| `id`      | `string` | `null`  | yes      | AccordionContent ID                                                 |
+| `class`   | `string` | `null`  | no       | Additional class name                                               |
+| `labelId` | `string` | `null`  | yes      | AccordionHeader ID                                                  |
+| `parent`  | `string` | `null`  | no       | A parent element selector that ensures that only one item is opened |
+| `isOpen`  | `string` | `false` | no       | Whether is item selected                                            |
```

Comment:
Same as above, word order and `expanded` work better:

```suggestion
| `isOpen`  | `string` | `false` | no       | Whether the item is expanded                                            |
```

Reply:
fixed